packages/expo-insights/ios/InsightsModule.swift

```diff
@@ -38,70 +40,70 @@ public final class InsightsModule: Module {
-  let endpointUrl = "https://i.expo.dev/v1/c/\(projectId)"
+  /**
+   Sends an event with the given name and data.
+   */
+  private func dispatchEvent(projectId: String, eventName: String, data: [String: String?]) async throws {
+    let endpointUrl = "https://i.expo.dev/v1/c/\(projectId)"
 
-  guard var urlComponents = URLComponents(string: endpointUrl) else {
-    log.warn("Insights: The URL for the HTTP endpoint is invalid: \(endpointUrl)")
-    return
-  }
-  var queryItems: [URLQueryItem] = []
+    guard var urlComponents = URLComponents(string: endpointUrl) else {
+      log.warn("Insights: The URL for the HTTP endpoint is invalid: \(endpointUrl)")
+      return
+    }
+    var queryItems: [URLQueryItem] = []
 
-  for (key, value) in data {
-    queryItems.append(
-      URLQueryItem(name: key, value: value)
-    )
-  }
+    for (key, value) in data {
+      queryItems.append(
+        URLQueryItem(name: key, value: value)
+      )
+    }
 
-  urlComponents.queryItems = queryItems
+    urlComponents.queryItems = queryItems
 
-  guard let url = urlComponents.url else {
-    log.warn("Insights: Cannot create an URL instance from the given query: \(urlComponents.query)")
-    return
-  }
-  var request = URLRequest(url: url)
+    guard let url = urlComponents.url else {
+      log.warn("Insights: Cannot create an URL instance from the given query: \(urlComponents.query ?? "Null query")")
+      return
+    }
+    var request = URLRequest(url: url)
 
-  request.httpMethod = "GET"
+    request.httpMethod = "GET"
 
-  let (_, response) = try await URLSession.shared.data(for: request)
+    let (_, response) = try await URLSession.shared.data(for: request)
 
-  guard let response = response as? HTTPURLResponse else {
-    log.warn("Insights: Unexpectedly the response is not of HTTPURLResponse type")
-    return
-  }
-  guard (200...299).contains(response.statusCode) else {
-    log.warn("Insights: Server responded with status code \(response.statusCode) for event \(eventName)")
-    return
+    guard let response = response as? HTTPURLResponse else {
+      log.warn("Insights: Unexpectedly the response is not of HTTPURLResponse type")
+      return
+    }
+    guard (200...299).contains(response.statusCode) else {
+      log.warn("Insights: Server responded with status code \(response.statusCode) for event \(eventName)")
+      return
+    }
   }
-}
 
-/**
- Gets the project ID from the manifest.
- */
-private func getProjectId(manifest: [String: Any]) -> String? {
-  let extra = manifest["extra"] as? [String: Any]
-  let eas = extra?["eas"] as? [String: Any]
+  /**
+   Gets the project ID from the manifest.
+   */
+  private func getProjectId(manifest: [String: Any]) -> String? {
+    let extra = manifest["extra"] as? [String: Any]
+    let eas = extra?["eas"] as? [String: Any]
 
-  return eas?["projectId"] as? String
-}
+    return eas?["projectId"] as? String
```

Comment:
looks like this uses tabs instead of spaces?

Reply:
Interesting -- it did pass Swiftlint though